### PR TITLE
[IN-1876] ignore xclogparser install in case of 9.4 and 10.3

### PR DESCRIPTION
**Pull request by lmesz-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Review by fehersanyi-bitrise — APPROVED** [review]
lgtm

File: roles/weekly-update-shared/tasks/main.yml [hunk 1]
```diff
@@ -92,6 +92,18 @@
...
+#TODO: ignore from the items list
+- name: Register Xcode version
+  shell: bash -l -c "$(/usr/bin/xcodebuild -version | head -n 1)"
+  register: xcode_version
+
+- name: Install xclogparser
+  homebrew:
+    name: xclogparser
+    state: present
+  when: xcode_version.stdout != "Xcode 9.4" || xcode_version.stdout != "Xcode 10.3"
```
Comment (MrBlaise) [inline roles/weekly-update-shared/tasks/main.yml:105]: This should be an *and* (&&) instead of an *or* (||)

Comment (lmesz-bitrise) [reply roles/weekly-update-shared/tasks/main.yml:105]: ?

**Review by MrBlaise — DISMISSED** [review]

**Review by peterheja-bitrise — DISMISSED** [review]
LGTM

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Review by MrBlaise — APPROVED** [review]

**Review by fehersanyi-bitrise — APPROVED** [review]

**Review by MrBlaise — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
lgtm

**Review by peterheja-bitrise — CHANGES_REQUESTED** [review]
explain `&& v`

**Comment by lmesz-bitrise** [comment]
> explain `&& v`

Ohh sometimes I mark my changes with that symbol :) /sarcasm
Nice catch, Sorry I left it there accidentally. :)

**Review by MrBlaise — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request closed without merging**